packages/hw-app-ada/test/src/current/integration/getExtendedPublicKey.js

```diff
@@ -0,0 +1,78 @@
+import { expect } from "chai";
+import pathDerivations from "../../pathDerivations";
+
+import { getAda, pathToArray } from "../../utils";
+
+describe("getExtendedPublicKey", async () => {
+  let ada = {};
+
+  beforeEach(async () => {
+    ada = await getAda();
+  });
+
+  afterEach(async () => {
+    await ada.t.close();
+  });
+
+  it("Should successfully get extended public key", async () => {
+    const der1 = pathDerivations["44'/1815'/1'"];
+    const der2 = pathDerivations["44'/1815'/1'/0/12'"];
+    const der3 = pathDerivations["44'/1815'/1'/0/10'/1/2/3"];
+
+    const result1 = await ada.getExtendedPublicKey(pathToArray(der1.path));
```

Comment:
This whole function seems to be too repetitive ...
Why not extract common logic to a lambda?